### PR TITLE
Feature request: Button to download SWTI data

### PR DESCRIPTION
**Pull request by BobTorgerson**
This provide a new feature that allows for the download of the last two years worth of SWTI data in CSV format.

To review:
- New button appears above graph stating: Download data
- Clicking on button results in a CSV file being created containing the last 2-years worth of SWTI data

### CODE REVIEW

File: assets/99_app.css [hunk 1]
```diff
@@ -11,6 +11,7 @@ h1, h2, h3, h4, h5, h6 {
 
 .content {
 	margin: 0 auto;
+	max-width: 40em;
```
Comment (brucecrevensten) [inline assets/99_app.css]: Oops good catch cleaning the CSS!  My fault before!!

File: gui.py [hunk 2]
```diff
@@ -100,10 +101,17 @@ def wrap_in_section(content, section_classes="", container_classes="", div_class
 )
 
 
-# Index as a bar chart
+# Index as a scatter chart
```
Comment (brucecrevensten) [inline gui.py]: Fixing incorrect comments: gold star ⭐️

**Review by brucecrevensten — CHANGES_REQUESTED** [review]
This is tight!  Good coding!  I like it!

Do we need the file which is to be downloaded added to the repo...?  Or add that to gitignore?

I'm not clear on what permissions need to be managed to allow the server to write this file...?

One direct thing that needs to be fixed -- rows like `0,2019-05-04,1.8019999999999998,25` are in that generated CSV.  *Index values must be rounded to 2 digits after the decimal*.  I kinda spotted this in the app but was like "eh good enough" -- it's not good enough with the download available!

**Comment by BobTorgerson** [comment]
This PR will close issue #9

**Pull request merged**